test/combine.js

```diff
@@ -35,3 +35,33 @@ test('nested combine', function (t) {
   t.equal(sockets.no[0](), false)
   t.end()
 })
+
+test('sequential combination of nested modules', function (t) {
```

Comment:
i'm not sure i understand, how is this testing the sequential ordering? wouldn't it pass even if you switched the order?